src/middleware.ts

```diff
@@ -3676,38 +3676,6 @@ const DEVELOPER_DOCS_REDIRECTS: Redirect[] = [
     from: '/frontend/development-server/',
     to: '/development-infrastructure/frontend-development-server/',
   },
-  {
-    from: '/backend/',
-    to: '/api-server/',
-  },
-  {
-    from: '/backend/api/',
-    to: '/api-server/api/',
-  },
-  {
-    from: '/backend/api/basics/',
-    to: '/api-server/api/basics/',
-  },
-  {
-    from: '/backend/api/design/',
-    to: '/api-server/api/design/',
-  },
-  {
-    from: '/backend/api/concepts/',
-    to: '/api-server/api/concepts/',
-  },
-  {
-    from: '/backend/api/public/',
-    to: '/api-server/api/public/',
-  },
-  {
-    from: '/backend/api/checklist/',
-    to: '/api-server/api/checklist/',
-  },
-  {
-    from: '/backend/api/serializers/',
-    to: '/api-server/api/serializers/',
-  },
```

Comment:
I would not delete, but revert them now. Meaning, the URLs with `/api-server/...` are now known to the internet, and should be added as redirects to the `/backend/...` URLs